src/main/java/uk/gov/hmcts/reform/managecase/api/controller/V1.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.managecase.api.controller;
+
+@SuppressWarnings("PMD.ShortClassName")
+public final class V1 {
+    private V1() {
+    }
+
+    public final class MediaType {
+
+        public static final String CASE_ASSIGNMENT_RESPONSE = "application/vnd.uk.gov.hmcts.ccd.manage-case-assignment"
+            + ".assign-access-within-organisation-response-payload.v1+json;charset=UTF-8";
```

Comment:
@mario-paniccia had later brought to my attention that the programme guidelines recommends not using vendor-specific media types for the first version of an api operation. Could we change this accordingly, please?

Reply:
ok, it's just normal application/json..?

Reply:
Can you update LLD please

Reply:
Sure. LLD Update done.